### PR TITLE
Update support for storage accounts

### PR DESCRIPTION
**Pull request by jthake-msft**

### CODE REVIEW

**Comment by jthake-msft** [comment]
Need to confirm whether we support v1 still?

**Comment by gladyscodes** [comment]
> Need to confirm whether we support v1 still?

I found that on Feb 29, 2024 Azure Data Lake Storage Gen1 will be retired. For more information, see the
 [official announcement](https://azure.microsoft.com/updates/action-required-switch-to-azure-data-lake-storage-gen2-by-29-february-2024/). 
People using Azure Data Lake Storage Gen1, need to make sure to migrate to Azure Data Lake Storage Gen2 prior to that date.

**Review by gladyscodes — APPROVED** [review]

File: concepts/data-connect-datasets.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Sinks are the output location that Azure Synapse or Azure Data Factory uses to p
 
 - [Azure Data Lake Storage Gen1](/azure/data-lake-store/data-lake-store-overview)
 - [Azure Data Lake Storage Gen2](/azure/storage/blobs/data-lake-storage-introduction)
-- [Azure Storage Blob](/azure/storage/blobs/storage-blobs-overview)
+- [Azure Storage Blob](/azure/storage/blobs/storage-blobs-overview) (only supported with Copy Data activity)
```
Comment (Danielabom) [inline concepts/data-connect-datasets.md:113]:
```suggestion
- [Azure Storage Blob](/azure/storage/blobs/storage-blobs-overview) (only supported with copy data activity)
```

**Comment by Danielabom** [comment]
Contacted author via Teams.

**Pull request closed without merging**